jquery.dataTables/jquery.dataTables-tests.ts

```diff
@@ -261,7 +262,7 @@ $(document).ready(function () {
         {
             ajax: {
                 data: ajaxDataFunc,
-                dataSrc: function (data) { },
+                dataSrc: function (data: any) { },
```

Comment:
there should be a contextual type flowing from the left hand side. i.e. `Settings.ajax` in this case, since `config` is defined as a `Setting`. 

The file was passing before using no-implicit-any i assume, so not sure why this is needed.